frontend/src/components/Shared/Navbar/Navbar.jsx

```
@@ -255,6 +229,56 @@ const NavigationBar = () => {
+            </nav>
+
+            {/* User Controls */}
+            {renderUserControls()}
+            
+            {/* Mobile menu button */}
+            <button 
+              data-mobile-toggle
+              onClick={() => setIsMobileMenuOpen(!isMobileMenuOpen)}
+              className="md:hidden text-gray-300 hover:text-white focus:outline-none"
+              aria-label="Toggle menu"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)